### ISSUE TITLE
Just wanted to say that...

### ISSUE THREAD

**Issue by mmazzarolo** [infinitered/ignite#11]
...this boilerplate is awesome, thank you for sharing it guys :+1:

**Comment by GantMan** [comment]
:+1: !  It's our plan to only make it better.   I hope to see you on the list of contributors :)

**Comment by skellock** [comment]
Best issue ever.

**Comment by imkmf** [comment]
Yes, it's really great! I'm really liking the Saga approach and it's had a huge impact on this app rewrite I'm doing. <3

**Comment by steve21124** [comment]
just want to add... ignite is super great project and very active. good work and keep churning...

**Comment by netuoso** [comment]
Unconventional comment on an unconventional issue.

GantMan is a weezard

**Comment by pekkanikolaus** [comment]
Seconded. This thing is great. React Native offers so little in terms of structure. Having to learn this entire new ecosystem AND worrying about how to best structure things long term is just too much. It’s great to have a battle-tested boilerplate to help getting into things. Thanks!

**Comment by skellock** [comment]
Cheers all.  Closing this with a warm & fuzzy feeling.

**Comment by jamonholmgren** [comment]
So I hate to re-open issues, but I think this one will never be resolved. So this issue is henceforth permanently open.

**Comment by bubuzzz** [comment]
I totally agree with this. This package is totally awesome. Everything is there and it just works. Everything is kept at a balanced level, no more, no less so it is easy to move with a fast momentum. So in love with it !!!!!

**Comment by jamonholmgren** [comment]
Thank you for the kind words @bubuzzz! It's really encouraging and motivating to hear that.

**Comment by alexakasanjeev** [comment]
Hi, I was wondering what it takes to get my boilerplate code get listed in ignite.

I have wrote a React Native boiler plate code that is based on Atomic design and uses redux, react-navigation, storybook, redux saga.
[Boiler plate code](http://github.com/alexakasanjeev/arcn)

**Comment by kevinvangelder** [comment]
@alexakasanjeev Only official Infinite Red boilerplates are included in the CLI by default, but you're welcome to open a PR to add yours to the list of 3rd party boilerplates here: https://github.com/infinitered/ignite/blob/master/BOILERPLATES.md

**Comment by LalitDWS** [comment]
I'm experiencing difficulty cloning this repository. I've generated an SSH key and attempted to clone it, but I'm encountering an issue. The error message is as follows:

ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by LalitDWS** [comment]
> @LalitDWS sounds like you're having issues with connecting properly to your Github account. Did you set up a personal token?

Yes, I've set up a personal token, but I'm still encountering issues. Any additional steps or troubleshooting suggestions you could provide?
